lib/circuitry/config/subscriber_settings.rb

```diff
@@ -0,0 +1,26 @@
+require 'virtus'
+require 'circuitry/config/shared_settings'
+
+module Circuitry
+  module Config
+    class SubscriberSettings
+      include Virtus::Model
+      include SharedSettings
+
+      attribute :queue_name, String
+      attribute :dead_letter_queue_name, String
+      attribute :visibility_timeout, Integer, default: 30 * 60
+      attribute :max_retry_count, Integer, default: 8
+      attribute :lock_strategy, Object, default: ->(_page, _attribute) { Circuitry::Locks::Memory.new }
+
+      def dead_letter_queue_name
+        super || "#{queue_name}-failures"
```

Comment:
Extreme edge case, but if `queue_name` isn't set for any reason, I wonder if we should clean this up with a value like `[queue_name, 'failures'].compact.join('-')`.  Probably overkill, but I'll leave the comment here regardless.